forwards_paper.tex

```diff
@@ -238,29 +238,25 @@ \subsection*{Simulation benchmarks}
 and the Linux \texttt{time} command to record run time and peak memory usage of each replicate.
 Our simulation scripts separately recorded the time spent in various steps of each run.
 
+\krt{currently, we are not doing anything with the detailed timings.  I think that is fine, and that we can delete last
+sentence of preceding paragraph.}
```

Comment:
sounds good: can comment it out to remind ourselves how it worked in case someone does something with the detailed timings.